src/js/components/FileInput/stories/Disabled.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { Box, FileInput, Grommet } from 'grommet';
+
+export const Disabled = () => (
+  <Grommet
```

Comment:
Since this has a custom theme, can you place it under a CustomThemed folder like your Anchor story?